app.rb

```diff
@@ -0,0 +1,79 @@
+require 'sinatra'
+require 'data_mapper'
+require 'rack-flash'
+
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, "postgres://localhost/chitter_#{env}")
+
+require './lib/cheet'
+require './lib/user'
+
+DataMapper.finalize
+DataMapper.auto_upgrade!
+
+enable :sessions
+set :session_secret, 'encryption key'
+use Rack::Flash
+use Rack::MethodOverride
+
+get '/' do
+  @cheets = Cheet.all
```

Comment:
could you do this so they're definitely in the correct chronological order. Right now they're just i the same order as they appear in the database which might not always be chronological.